crates/l2/docs/sequencer.md

```diff
@@ -88,7 +89,8 @@ The following environment variables are available to configure the Proposer cons
   - `l1_private_key`: Its private key.
   - `listen_ip`: IP to listen for proof data requests.
   - `listen_port`: Port to listen for proof data requests.
-  - `dev_mode`: Whether `dev_mode` is activated or not.
+  - `proof_send_interval_ms`: Interval in which the committer wakes up and send commitments to the L1
+  - `dev_mode`: Whether `dev_mode` is activated or not. If it is activated, proofs won't be verified just executed. This can be seen on the OnChainProposer contract, specifically on the verify function, proofs will be ignored.
```

Comment:
> Whether `dev_mode` is activated or not. If it is activated, proofs won't be verified just executed. This can be seen on the OnChainProposer contract, specifically on the verify function, proofs will be ignored.

Whether `dev_mode` is activated or not. If it is activated, the prover won't actually generate proofs, but just execute the blocks it is given, and the proof sender will send empty proofs to the L1. The L1, in turn, will ignore proofs in dev mode.